common/lib/xmodule/xmodule/js/src/capa/display.js

```diff
@@ -377,406 +441,461 @@
+                window.SR.readText(gettext('Answers to this problem are now shown. Navigate through the problem to review it with answers inline.')); // eslint-disable-line max-len
+                that.scroll_to_problem_meta();
             });
         };
 
-        Problem.prototype.clear_all_notifications = function () {
+        Problem.prototype.clear_all_notifications = function() {
             this.submitNotification.remove();
             this.gentleAlertNotification.hide();
-            return this.saveNotification.hide();
+            this.saveNotification.hide();
         };
 
-        Problem.prototype.gentle_alert = function (msg) {
-            edx.HtmlUtils.setHtml(this.el.find('.notification-gentle-alert .notification-message'), edx.HtmlUtils.HTML(msg));
+        Problem.prototype.gentle_alert = function(msg) {
+            edx.HtmlUtils.setHtml(
+                this.el.find('.notification-gentle-alert .notification-message'),
+                edx.HtmlUtils.HTML(msg)
+            );
             this.clear_all_notifications();
             this.gentleAlertNotification.show();
-            return this.gentleAlertNotification.focus();
+            this.gentleAlertNotification.focus();
         };
 
-        Problem.prototype.save = function () {
+        Problem.prototype.save = function() {
             if (!this.submit_save_waitfor(this.save_internal)) {
-                return this.disableAllButtonsWhileRunning(this.save_internal, false);
+                this.disableAllButtonsWhileRunning(this.save_internal, false);
             }
         };
 
-        Problem.prototype.save_internal = function () {
-            var _this = this;
+        Problem.prototype.save_internal = function() {
+            var that = this;
             Logger.log('problem_save', this.answers);
-            return $.postWithPrefix("" + this.url + "/problem_save", this.answers, function (response) {
+            return $.postWithPrefix('' + this.url + '/problem_save', this.answers, function(response) {
                 var saveMessage;
                 saveMessage = response.msg;
                 if (response.success) {
-                    _this.el.trigger('contentChanged', [_this.id, response.html]);
-                    edx.HtmlUtils.setHtml(_this.el.find('.notification-save .notification-message'), edx.HtmlUtils.HTML(saveMessage));
-                    _this.clear_all_notifications();
-                    _this.saveNotification.show();
-                    return _this.focus_on_save_notification();
+                    that.el.trigger('contentChanged', [that.id, response.html]);
+                    edx.HtmlUtils.setHtml(
+                        that.el.find('.notification-save .notification-message'),
+                        edx.HtmlUtils.HTML(saveMessage)
+                    );
+                    that.clear_all_notifications();
+                    that.saveNotification.show();
+                    return that.focus_on_save_notification();
```

Comment:
With all the other returns that we removed do we need this one still?  Coffeescript seemed to add a return to the last line of every method/condition.